packages/framework-provider-kubernetes-infrastructure/src/infrastructure/dapr-manager.ts

```diff
@@ -9,7 +9,15 @@ import * as Mustache from 'mustache'
 import { BoosterConfig, Logger } from '@boostercloud/framework-types'
 import { getProjectNamespaceName } from './utils'
 import { scopeLogger } from '../helpers/logger'
+import { safeLoad } from 'js-yaml'
 
+interface stateStoreYaml {
```

Comment:
All types should be capitalized
```suggestion
interface StateStoreYaml {
```